MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -3319,10 +3379,10 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "com.mapbox.Example-CarPlay";
+				PRODUCT_BUNDLE_IDENTIFIER = "com.Mapbox.CarPlay-Example";
```

Comment:
Bundle of `Example-CarPlay` target had to be updated to use updated provisioning profile.